### ISSUE TITLE
Google search results showing Drupal favicon

### ISSUE THREAD

**Issue by blairlearn** [NCIOCPL/cgov-digital-platform#3893]
### Issue description
Google search results are showing the wrong icon.

> **ESTIMATE** TBD

### Steps to reproduce the issue

1. Go to https://www.google.com
2. Search for `What is cancer`
3. Scroll to results showing links to www.cancer.gov

### What's the expected result?
- Links should be accompanied by the NCI favicon 
![image](https://github.com/NCIOCPL/cgov-digital-platform/assets/21063180/a61001f1-6af2-4abd-8b3f-4602d4021c6b)

### What's the actual result?
- Links are accompanied by the Drupal favicon
![image](https://github.com/NCIOCPL/cgov-digital-platform/assets/21063180/7d91c13b-b76a-451c-a05b-5bf555c0f9de)

### Additional details / screenshot
In the generated page HTML, a `<link>` to the  Drupal favicon appears _before_ the ones for the NCI favicon.  Potential fixes are to either suppress the Drupal favicon, or render the NCI link elements first.

![image](https://github.com/NCIOCPL/cgov-digital-platform/assets/21063180/76c08bda-f333-43b6-bed8-c7d54c2599d6)

![image](https://github.com/NCIOCPL/cgov-digital-platform/assets/21063180/8386047e-af46-4ce2-91bd-c8c3ba84cd22)

### How to fix?

1. Update our theme settings (not sure if we need to do both [cgov](https://github.com/NCIOCPL/cgov-digital-platform/blob/develop/docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/config/install/cgov_common.settings.yml) and [ncids_trans](https://github.com/NCIOCPL/cgov-digital-platform/blob/develop/docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/config/install/ncids_trans.settings.yml)) to [specify the default favicon](https://drupal.stackexchange.com/questions/12834/change-default-favicon-programmatically/184852#184852).

````yml
favicon:
  use_default: false
  path: 'profiles/custom/cgov_site/themes/custom/cgov/static/images/design-elements/icons/favicons/favicon.ico'
````

2. Create the appropriate update hooks to change existing installs of the theme and update those two fields to the values specified above.

3. Reroute `/favicon.ico` to our custom favicon. Update [.htaccess](https://github.com/NCIOCPL/cgov-digital-platform/blob/5a6d6611ba586353c6f344b9bc9d13661a0b9c8a/docroot/.htaccess#L148) and extract the favicon line to a different block above it's current one. Something like:

````
RewriteCond %{REQUEST_URI} ^/favicon.ico$
RewriteRule ^favicon.ico$ /profiles/custom/cgov_site/themes/custom/cgov/static/images/design-elements/icons/favicons/favicon.ico [L]
````

4. Remove the `<link rel='shortcut icon'>` tags from our [twig templates](https://github.com/search?q=repo%3ANCIOCPL%2Fcgov-digital-platform+%22shortcut+icon%22+language%3ATwig&type=code&l=Twig). No need to specify them in the code if Drupal is going to serve the correct icon.

5. Remove the favicon rule from the `WWW` Akamai property. We don't need it there if we handle this in Drupal. Plus here it applies to all the sites.

### Related Tickets
* Issue #2647
* Issue #2646

**Comment by andyvanavery31** [comment]
As of 08FEB2024 - there is an Akamai work around that makes it appear as though it has been resolved. This is not a fix, and this still needs to be fixed.

**Comment by andyvanavery31** [comment]
@blairlearn can you confirm if this is still an issue? If yes, this should be dropped to the lowest priority for traceability. If no, this ticket can be closed.

**Comment by blairlearn** [comment]
Fixed in c617ca3f3111946b219b151e854650c2714b9436
(PR #4337 as part of issue #4300) 


### Verified October 8, 2024

Google now displays the correct logo for sites hosted on the digital platform.

![Screenshot 2024-10-08 at 5 08 22 PM](https://github.com/user-attachments/assets/f6c784e2-b613-4908-a9ef-f648d0440cab)
![Screenshot 2024-10-08 at 5 09 13 PM](https://github.com/user-attachments/assets/cd4a46b0-1d3f-48e3-b7d6-efa186cc98ce)
![Screenshot 2024-10-08 at 5 11 11 PM](https://github.com/user-attachments/assets/83dd1138-2276-4991-ba24-ec77974262cd)
![Screenshot 2024-10-08 at 5 11 57 PM](https://github.com/user-attachments/assets/7bd6a88e-03c1-4c94-8c08-cbd021adefe5)

Related: As part of the same fix, we are also able to customize the logo.
![Screenshot 2024-10-08 at 5 13 48 PM](https://github.com/user-attachments/assets/6768dec1-0099-4772-babd-aca8f015929a)
